### DISCUSSION TITLE
how to use slot3/ slot 4 for direct_XIP mode

### DISCUSSION THREAD

**Discussion by RajT33** [mcu-tools/mcuboot#2090] [General]
hello , 
I am using STM32H750b_dk board with mcuboot in direct_XIP mode.
currently i am using slot1 for app image, but i want to use different slot for different app images.
when i try to change from slot1 to slot3/slot4 for app image it is not working.
may i know what config i am missing to change.

**Comment by nordicjm** [comment]
Not supported

**Reply by RajT33** [reply]
is it for only STM32H750b_dk board or it is not supported at all?

**Reply by nordicjm** [reply]
Not supported at all, there are 2 slots per image

**Reply by RajT33** [reply]
thank you for supporting.
